libc-bottom-half/headers/public/__header_dirent.h

```diff
@@ -11,6 +11,12 @@
 #define DT_REG __WASI_FILETYPE_REGULAR_FILE
 #define DT_UNKNOWN __WASI_FILETYPE_UNKNOWN
 
+#define IFTODT(x) (__wasilibc_iftodt(x))
+#define DTTOIF(x) (__wasilibc_dttoif(x))
+
+int __wasilibc_iftodt(int x);
+int __wasilibc_dttoif(int x);
```

Comment:
Where are these defined?

Reply:
I had forgotten to `git add` these. Fixed now!